index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
+                <div class="laptop-img">
+                    <img src="images/laptop4.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
+    
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop5.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
+    </section>
+    <section class="about">
+        <div>
+            <h2 class="headings">About</h2>
+            <div class="content">
+               <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+               <button class="button download">Get CV
+                <div class="icon"></div>
+               </button>
+            </div>
+        </div>
+    </section>
+    <section class="competency">
+        <div class="grid-container">
+            <div class="language grid-item">
+                <h3>Languages</h3>
+                <ul>
+                    <li>Javascript(ES5/ES6)</li>
+                    <li>Ruby</li>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                </ul>
+            </div>
+            <div class="frameworks grid-item">
+                <h3>Frameworks</h3>
+                <ul>
+                    <li>React.js</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capibara</li>
+                </ul>
+            </div>
+            <div class="skills grid-item">
+                <h3>Skills</h3>
```

Comment:
- [ ] Please note that the The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` as given in the project requirements 🤓⛏

  > _Please use `ul` and `li` for categories in "About myself" section. Languages, Frameworks, and Skills should be list items_

- [ ] Also, please note that the button(Get my CV) and technology categories are aligned. Please fix this. See the images below for more clarity 👇 Also, please add some more space between the button and technology categories and change the text from `Get my CV` 💪

Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/236018784-7831e988-254a-46a2-83dc-508af51e0bda.png) | ![image](https://user-images.githubusercontent.com/106119105/236018635-6f7fb2ff-b055-41c8-bed7-e1afdd968d6c.png)

- [ ] Similarly, please make sure to align the following according to the given Figma design as well 👇 and reduce the `font-weight` of languages such as HTML, CSS etc

Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/236019636-9d9e3ce5-0606-4255-8985-28401321312f.png) | ![image](https://user-images.githubusercontent.com/106119105/236019543-ec37c374-b59b-48ee-a501-62faabda5cfc.png)